### PR TITLE
SRCH-135

### PR DESCRIPTION
**Pull request by scott-lewis-nsidc**
Rubocop updates

### CODE REVIEW

File: lib/search_solr_tools/errors/harvest_error.rb [hunk 1]
```diff
@@ -10,55 +12,65 @@ class HarvestError < StandardError
...
+        Helpers::HarvestStatus::HARVEST_NO_DOCS => ERRCODE_SOURCE_NO_RESULTS,
+        Helpers::HarvestStatus::HARVEST_FAILURE => ERRCODE_SOURCE_HARVEST_ERROR,
+        Helpers::HarvestStatus::INGEST_ERR_INVALID_DOC => ERRCODE_DOCUMENT_INVALID,
+        Helpers::HarvestStatus::INGEST_ERR_SOLR_ERROR => ERRCODE_INGEST_ERROR,
+        Helpers::HarvestStatus::OTHER_ERROR => ERRCODE_OTHER
       }.freeze
 
       # If code is -1, it means display all error codes
       def self.describe_exit_code(code = -1)
+        code_list = code_to_list(code)
```
Comment (scott-lewis-nsidc) [inline lib/search_solr_tools/errors/harvest_error.rb:39]: This one is just splitting the method into two to help with one of the rubocop checks (I think it was the ABC branching, but I can't remember for sure right now).

File: lib/search_solr_tools/helpers/iso_namespaces.rb [hunk 2]
```diff
@@ -7,25 +9,25 @@ def self.namespaces(doc = nil)
       end
 
       ISO_NAMESPACES = {
-        'atom'   => 'http://www.w3.org/2005/Atom',
```
Comment (scott-lewis-nsidc) [inline lib/search_solr_tools/helpers/iso_namespaces.rb:10]: This was done by the rubocop "autocorrect".  I can probably move it back and update a rubocop setting to use a value-aligned style.

File: lib/search_solr_tools/harvesters/nsidc_json.rb [hunk 3]
```diff
@@ -81,13 +82,13 @@ def docs_with_translated_entries_from_nsidc
...
             docs << { 'add' => { 'doc' => @translator.translate(fetch_json_from_nsidc(id)) } }
-          rescue => e
+          rescue StandardError => e
             puts "Failed to fetch #{id} with error #{e}: #{e.backtrace}"
             failure_ids << id
           end
         end
 
-        { num_docs: all_docs.size, add_docs: docs, failure_ids: failure_ids }
+        { num_docs: all_docs.size, add_docs: docs, failure_ids: }
```
[the remark this answers is not in the document]
Comment (juliacollins) [reply lib/search_solr_tools/harvesters/nsidc_json.rb:91]: > Does the latest Ruby magically assigned the value of the variable `failure_ids` to the hash key `failure_ids`?

Trying a comment to see if it sticks...

**Review by juliacollins — APPROVED** [review]

**Review by juliacollins — APPROVED** [review]
Trying another approval to see if Slack gets the update.

**Review by juliacollins — APPROVED** [review]
One last approval after re-requesting review.

**Pull request merged**